SCC/example.cpp

```diff
@@ -0,0 +1,124 @@
+//NAME OF QUESTION: CAPCITY - Capital City (SPOJ)
```

Comment:
The example doesn't seem to use the SCC code snippet that you added.
The goal here is to show how the existing template was used to get a solution accepted.